DigitalLearningSolutions.Web.Tests/Controllers/LearningMenu/LearningMenuControllerTests.cs

```diff
@@ -626,6 +626,101 @@ public void Sections_should_redirect_to_post_learning_assessment_if_there_is_dia
                 .WithRouteValue("sectionId", sectionId);
         }
 
+        [Test]
+        public void Sections_should_redirect_to_diagnostic_assessment_if_only_diagnostic_in_section()
+        {
+            // Given
+            const int customisationId = 123;
+            const int sectionId = 456;
+            var expectedSectionContent = SectionContentHelper.CreateDefaultSectionContent(
+                diagAssessPath: "some/diagnostic/path.html",
+                plAssessPath: null,
+                consolidationPath: null,
+                otherSectionsExist: true
+            );
+            expectedSectionContent.DiagnosticStatus = true;

```

Comment:
Why not set this as part of creating the section content? 🙂

Reply:
This value is not exposed as part of the constructor, it is set as the list of tutorials is processed in the multi-map part of the query call in the service. I could amend this helper method to set these values after content has been created if you think that would make it clearer? 🙂

Reply:
I'm pretty sure it is part of the constructor, it just gets updated as part of the multimap? 🙂

Reply:
Yes it is, sorry about that! Fixed in ee9c443 🙂